guidelines/technical-guidelines/03-custom-resource-definitions.md

```diff
@@ -10,19 +10,95 @@ If you use a third-party CRD, apply the [location and file name](#custom-resourc
 
 ## Location and file name
 
-Place the Kyma CRDs in the Helm chart folder of the component that they are part of.
+Place the Kyma CRDs in the `cluster-essentials` Helm chart folder uder `files` subdirectory.
 
-When creating file names use the `names.singular` format, for example `crontab.crd.yaml`. To learn more, see the [Naming](#custom-resource-definition-custom-resource-definition-naming) section. Include other names or terms in the file names to differentiate them from other CRDs, for example `crontab-v1.crd.yaml`. In the file name, do not include words which appear in the file's path. For example, `resources/core/charts/batch/templates/batch-crontab.crd.yaml` is not compliant because the word "batch" appears both in the file name and the path.
+When creating file names use the `names.singular` format, for example `crontab.crd.yaml`. To learn more, see the [Naming](#custom-resource-definition-custom-resource-definition-naming) section. Include other names or terms in the file names to differentiate them from other CRDs, for example `crontab-v1.crd.yaml`. In the file name, do not include words which appear in the file's path. For example, `resources/cluster-essentials/files/batch-crontab.crd.yaml` is not compliant because the word "batch" appears both in the file name and the path.
 
-To differentiate CRDs from other types of Kubernetes resource files and enable their installation during the eventual initialization step, end the file names with the `.crd.yaml` suffix and include `CustomResourceDefinition` or any subset of it. If a file name consists of several words, separate them with hyphens, and do not use capital letters.
+To differentiate CRDs from other types of Kubernetes resource files end the file names with the `.crd.yaml` suffix and include `CustomResourceDefinition` or any subset of it. If a file name consists of several words, separate them with hyphens, and do not use capital letters.
+
+You might encounter a problem when attempting to use a CRD in the Helm chart because it is not yet available in the Kubernetes cluster.
+
+## CRD ConfigMap
+
+During the initial phase of installation/upgrade, CRDs are mounted in `ConfigMap`. 
+For convenience maps bundle component CRDs and are located in the same file the installation/upgrade `Job` is located in (`resources/cluster-essentials/templates`).

```

Comment:
Maybe it would be good to add the link to the templates folder: https://github.com/kyma-project/kyma/tree/master/resources/cluster-essentials/templates?